diff --git a/test/fixtures/multiple-add-and-subtract.js b/test/fixtures/multiple-add-and-subtract.js
index 627c742..b3bb281 100644
--- a/test/fixtures/multiple-add-and-subtract.js
+++ b/test/fixtures/multiple-add-and-subtract.js
@@ -1,13 +1,15 @@
 const daysOfSun = 250,
-      foods = ['tzatziki','feta','gyros'],
-      islands = { inhabited: 170, total: 2000}
+      islands = { inhabited: 170, total: 2000},
+      weddingEvents = ['plate-throwing','dancing' ];
 
 function getFactAboutCountry(fact) {
   switch(fact) {
     case 'food':
-      return foods[1];
+      return foods[0];
     case 'islands':
-      return islands.inhabited
+      return islands.total;
+    case 'wedding':
+      return weddingEvents[0]
     default:
       return "These are fun facts!";
   }
